src/Microsoft.Health.Fhir.Api/Features/Bundle/BundleAwareJwtBearerHandler.cs

```diff
@@ -41,6 +58,7 @@ protected override async Task HandleForbiddenAsync(AuthenticationProperties prop
             if (_bundleHttpContextAccessor.HttpContext != null)
             {
                 _bundleHttpContextAccessor.HttpContext.Response.StatusCode = 403;
+                _auditHelper.LogExecuted(_bundleHttpContextAccessor.HttpContext, _claimsExtractor);
```

Comment:
When would this be logged?

Reply:
Yes, this is triggered each time a sub-request encounters the `403`. The code added to the `AuditLoggingFilterAttribute` is used filter out the audit logging in the case of a `403`.